xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/NotificationConfiguration.java

```diff
@@ -85,4 +86,14 @@ default int getAsyncPoolSize()
     {
         return 2;
     }
+
+    /**
+     * @return the hint of the component to be used for the email grouping strategy.
+     * @since 15.5RC1
+     */
+    @Unstable
+    default String getEmailGroupingStrategyHint()
+    {
+        return "";
```

Comment:
What is the meaning of empty string ?

Reply:
Mistake on my side, it should be "default" I'll fix it.